codegen/protocol-test-codegen-local/model/endpoints-string-array.smithy

```diff
@@ -0,0 +1,197 @@
+$version: "2.0"
```

Comment:
This is the Smithy model file provided by SEP that has prescribed test cases. Only edits made were putting on `@service` trait and `@restJson1` trait to the service shape, and adding `@http` traits to each of the operation shape with dummy values. This was done to "plug" this into existing protocol test codegen flow. Building the local protocol test gradle project generates Swift service code for this model, along with the endpoint tests.